src/test/java/io/reactivex/internal/operators/flowable/FlowableTakeLastOneTest.java

```diff
@@ -118,7 +118,9 @@ public void requestMore(long n) {
 
         @Override
         public void onStart() {
-            request(initialRequest);
+            if (initialRequest > 0) {
```

Comment:
hmm, why though? Isn't it normally validated by `SubscriptionHelper#validate()`?

Reply:
Yes, that's how I found this mistake. The intention wasn't to trigger the validation but simply not request upfront.

Reply:
Yes, only that default stacktrace printout of the validation error.

Reply:
I see

If you want I can make a JUnit Rule that'll by default check that there are no RxJavaPlugin errors and an annotation to allow errors on specific tests